pippo-demo/src/main/java/ro/fortsoft/pippo/demo/SimpleApplication.java

```diff
@@ -40,7 +40,7 @@ public void init() {
 
             @Override
             public void handle(Request request, Response response, RouteHandlerChain chain) {
-                response.send("Hello World");
+                response.text().send("Hello World");
```

Comment:
Maybe it's to much and redundant. I like the short version `response.send(myText)`. I think that we must promote always the simple version.